datalake/src/main/java/com/sequenceiq/datalake/service/sdx/ProvisionerService.java

```diff
@@ -175,7 +176,7 @@ public void startStackProvisioning(Long id, DetailedEnvironmentResponse environm
                 sdxClusterRepository.save(sdxCluster);
                 cloudbreakFlowService.saveLastCloudbreakFlowChainId(sdxCluster, stackV4Response.getFlowIdentifier());
                 LOGGER.info("Sdx cluster updated");
-            } catch (ClientErrorException e) {
+            } catch (ClientErrorException | InternalServerErrorException e) {
                 String errorMessage = webApplicationExceptionMessageExtractor.getErrorMessage(e);
                 LOGGER.info("Can not start provisioning: {}", errorMessage, e);
                 throw new RuntimeException("Can not start provisioning, client error happened on Cloudbreak side: " + errorMessage);
```

Comment:
The exception message is misleading

Reply:
is it better? `"Can not start provisioning, some error happened on Cloudbreak side: " + errorMessage`

Reply:
Cannot*

Reply:
https://www.lexico.com/grammar/cannot-or-can-not
someone prefer that format, what is the problem with that?

Reply:
"the first is much more usual". Users will see this and they would also think can not looks weird and unusual

Reply:
ok